content/docs/start/metrics-parameters-plots.md

```diff
@@ -141,7 +141,7 @@ which will be useful for comparison: parameters.
 
 ## Defining stage parameters
 
-It's pretty common for data science pipelines to include configuration files to
+It's pretty common for data science pipelines to include configuration files that 
```

Comment:
extra space here - should also add to checks?

Reply:
it should be part of the prettier already 🤔

Reply:
cc @jorgeorpinel ^^

Reply:
I don't think Prettier cares because sometimes leaving extra spaces means something in md.

Reply:
let's create a ticket and do some research on this

I think it should be part of the prettier. It should be able to recognize when it matters (rare cases)

Reply:
Or maybe it does actually. Anyway, this was rewrapped by Restyled and the whitespace trimmed. See https://github.com/iterative/dvc.org/pull/2508/files#diff-2eb7f6a5facb509ea8b85bed3befaa586d50ea0a25bb2fa4ed54ccd8fcc77708

Reply:
> let's create a ticket and do some research on this

I tried it locally and Prettier does remove extra spaces. Caught by the pre-commit hook

Reply:
It is still an issue that we didn't catch this and CI doesn't fail for the website

Reply:
We caught it. I merged the Restyled PR. CI was failing too (I'll double check in a draft PR).

Reply:
Yeah it was failing: https://app.circleci.com/pipelines/github/iterative/dvc.org/9147/workflows/25ac67af-6811-43ab-be48-2394769171d5/jobs/9204

I double checked with https://github.com/iterative/dvc.org/pull/2534 and it's failing too. We're good I think.